include/mitsuba/eradiate/oceanprops.h

```diff
@@ -572,11 +571,175 @@ std::tuple<Float, Float> cox_munk_crosswind_upwind(const Float& wind_speed) {
     return { sigma_cross_2, sigma_along_2 };
 }
 
+/**
+ * @brief Isotropic Mean Squared Slope *squared* as described by Cox and Munk
+ * (1954)
+ *
+ * @param wind_speed speed of wind at sea surface (mast height) [m/s].
+ * @return Float
+ * The isotropic wind mean square slpe squared
+ */
 template<typename Float>
 Float cox_munk_MMS(const Float& wind_speed) {
```

Comment:
Only seeing this now: this function name is not super expressive. How about `cox_munk_msslope_squared()`?